public/app/plugins/panel/barchart/bars.ts

```diff
@@ -12,8 +12,7 @@ import {
 } from '@grafana/schema';
 import { measureText, PlotTooltipInterpolator } from '@grafana/ui';
 import { formatTime } from '@grafana/ui/src/components/uPlot/config/UPlotAxisBuilder';
-
-import { preparePlotData2, StackingGroup } from '../../../../../packages/grafana-ui/src/components/uPlot/utils';
+import { preparePlotData2, StackingGroup } from '@grafana/ui/src/components/uPlot/utils';
```

Comment:
This import started causing a type error: there should not be direct file imports from one package into another, should always use the canonical package name instead